launcher/cog-launcher.c

```diff
@@ -1251,9 +1318,7 @@ cog_launcher_handle_local_options(GApplication *application, GVariantDict *optio
     }
 
     if (s_options.filter_path) {
-        WebKitWebsiteDataManager *data_manager = cog_launcher_get_web_data_manager(launcher);
-        g_autofree char          *filters_path =
-            g_build_filename(webkit_website_data_manager_get_base_cache_directory(data_manager), "filters", NULL);
+        g_autofree char          *filters_path = g_build_filename(g_get_user_cache_dir(), g_get_prgname(), "filters", NULL);
```

Comment:
It makes me a bit sad that we no longer have a “base directories for all cache/user data” that we can query WebKit for.

Reply:
Well, there's `webkit_website_data_manager_get_base_data_directory()` and `webkit_website_data_manager_get_base_cache_directory()`, but they return what you set, so when set to `NULL` to use the default you get `NULL`. I don't know why we designed it this way, I guess the idea is that either you set it yourself (and then you know it) or use the default (which is known too, but we can't change it in the future if apps rely on it). I thought about changing that in the new API, since you don't really set the values directly to the website data manager. I really prefer if the getters return the paths actually used, so only `NULL` in case of ephemeral session. What do you think @mcatanzaro ?

Reply:
Agree.